addon/modifiers/style.ts

```diff
@@ -3,7 +3,12 @@ import { dasherize } from '@ember/string';
 import { assert } from '@ember/debug';
 import { typeOf } from '@ember/utils';
 
-function isObject(o) {
+// Typing as `Partial<CSSStyleDeclaration>` does not work as some CSS property
+// names used in our tests (e.g. `font-size`) does not exist on CSStyleDeclaration.
+// Should investigate later why that's the case.
+type CSSStyles = { [key: string]: string | undefined };
```

Comment:
```suggestion
// Cannot be typed as `Partial<CSSStyleDeclaration>` because `CSSStyleDeclaration`
// interface does _not_ included dashed CSS property names. It only includes the
// camelCase version of a CSS property.
// https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1672
type CSSStyles = { [key: string]: string | undefined };
```